templates/about2expire.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+$this->data['header'] = sprintf(
+    'Your password will expire in %s %s',
+    $this->data['daysleft'],
+    $this->data['dayOrDays']
+);
+$this->data['autofocus'] = 'yesbutton';
+
+$this->includeAtTemplateBase('includes/header.php');
+
+?>
+<p>
+  The password for your <?= htmlentities($this->data['accountName']); ?>
+  account will expire on <?= htmlentities($this->data['expireOnDate']); ?>.
+</p>
+<p>
+  Would you like to update your password now?
+</p>
+
+<form action="<?= htmlspecialchars($this->data['formTarget']); ?>">
```

Comment:
I think we were discussing the possibility of just sticking with `htmlentities` unless you found some reason to specialize the use?